packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -47,6 +47,18 @@ export const SendModalView = ({ closeModal }: Props) => {
   const debounceRef = useRef<NodeJS.Timeout | null>(null);
   const [loading, setLoading] = useState(false);
 
+  useEffect(() => {}, [fields]);
+
+  const handleBack = (amount: string, address: string) => {
+    setSummaryModalOpen(false);
+
+    setFields((prevFields) => ({
+      ...prevFields,
+      amount: amount,
+      address: address,
+    }));
+  };
```

Comment:
```suggestion
  const handleBack = () => {
    setSummaryModalOpen(false);
  };
```
setFields is no need, as the user still in send model, the state has remained